Userland/Applications/Spreadsheet/HelpWindow.cpp

```diff
@@ -91,18 +91,17 @@ HelpWindow::HelpWindow(GUI::Window* parent)
             auto& doc = doc_option.as_object();
             const auto& name = url.fragment();
 
-            auto example_data_value = doc.get_or("example_data", JsonObject {});
-            if (!example_data_value.is_object()) {
+            if (!doc.has_object("example_data")) {
                 GUI::MessageBox::show_error(this, String::formatted("No example data found for '{}'", url.path()));
                 return;
             }
+            auto& example_data = doc.get("example_data").as_object();
```

Comment:
This is _three_ HashMap lookups, let's maybe tone it down?
An alternative API could be
```c++
auto example_data_ptr = doc.get_object("example_data");
if (!example_data_ptr)
    return;
auto& example_data = *example_data_ptr();
```